pypesto/optimize/load.py

```diff
@@ -82,6 +82,12 @@ def fill_result_from_history(
     if not optimize_options.history_beats_optimizer:
         return result
 
+    if isinstance(optimizer_history.history, Hdf5History):
+        if optimizer_history.history.message is not None:
+            result.message = optimizer_history.history.message
+        if optimizer_history.history.exitflag is not None:
+            result.exitflag = optimizer_history.history.exitflag
```

Comment:
```suggestion
        if (message := optimizer_history.history.message) is not None:
            result.message = message
        if (exitflag := optimizer_history.history.exitflag) is not None:
            result.exitflag = exitflag
```
Avoids opening the file a second time.

Reply:
uh nice, didn't know that one :)

Reply:
More info: https://realpython.com/python-walrus-operator/ (python >=3.8)